library/Icingadb/Util/Format.php

```diff
@@ -0,0 +1,275 @@
+<?php
+
+namespace Icinga\Module\Icingadb\Util;
+
+use DateTime;
+
+class Format
```

Comment:
Please name it `PerfDataFormat` and remove all unused methods. (e.g. `secondsByMonth`)